### ISSUE TITLE
2025-05-07, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#162]
# Today's Arxiv Papers

(1) [VITA-Audio: Fast Interleaved Cross-Modal Token Generation for Efficient Large Speech-Language Model](http://arxiv.org/abs/2505.03739v1), 2025-05-06

(2) [WebGen-Bench: Evaluating LLMs on Generating Interactive and Functional Websites from Scratch](http://arxiv.org/abs/2505.03733v1), 2025-05-06

(3) [Graph Drawing for LLMs: An Empirical Evaluation](http://arxiv.org/abs/2505.03678v1), 2025-05-06

(4) [Binding threshold units with artificial oscillatory neurons](http://arxiv.org/abs/2505.03648v1), 2025-05-06

(5) [PhysLLM: Harnessing Large Language Models for Cross-Modal Remote Physiological Sensing](http://arxiv.org/abs/2505.03621v1), 2025-05-06

(6) [DyGEnc: Encoding a Sequence of Textual Scene Graphs to Reason and Answer Questions in Dynamic Scenes](http://arxiv.org/abs/2505.03581v1), 2025-05-06

(7) [LlamaFirewall: An open source guardrail system for building secure AI agents](http://arxiv.org/abs/2505.03574v1), 2025-05-06

(8) [Say It Another Way: A Framework for User-Grounded Paraphrasing](http://arxiv.org/abs/2505.03563v1), 2025-05-06

(9) [A Comprehensive Survey of Large AI Models for Future Communications: Foundations, Applications and Challenges](http://arxiv.org/abs/2505.03556v1), 2025-05-06

(10) [A Hashgraph-Inspired Consensus Mechanism for Reliable Multi-Model Reasoning](http://arxiv.org/abs/2505.03553v1), 2025-05-06

(11) [STORY2GAME: Generating (Almost) Everything in an Interactive Fiction Game](http://arxiv.org/abs/2505.03547v1), 2025-05-06

(12) [Faster MoE LLM Inference for Extremely Large Models](http://arxiv.org/abs/2505.03531v1), 2025-05-06

(13) [Ruled by the Representation Space: On the University's Embrace of Large Language Models](http://arxiv.org/abs/2505.03513v1), 2025-05-06

(14) [BadLingual: A Novel Lingual-Backdoor Attack against Large Language Models](http://arxiv.org/abs/2505.03501v1), 2025-05-06

(15) [Augmenting Human Cognition through Everyday AR](http://arxiv.org/abs/2505.03492v1), 2025-05-06

(16) [am-ELO: A Stable Framework for Arena-based LLM Evaluation](http://arxiv.org/abs/2505.03475v1), 2025-05-06

(17) [Evaluation of LLMs on Long-tail Entity Linking in Historical Documents](http://arxiv.org/abs/2505.03473v1), 2025-05-06

(18) [Long-Short Chain-of-Thought Mixture Supervised Fine-Tuning Eliciting Efficient Reasoning in Large Language Models](http://arxiv.org/abs/2505.03469v1), 2025-05-06

(19) [Uncertainty-Aware Large Language Models for Explainable Disease Diagnosis](http://arxiv.org/abs/2505.03467v1), 2025-05-06

(20) [LogisticsVLN: Vision-Language Navigation For Low-Altitude Terminal Delivery Based on Agentic UAVs](http://arxiv.org/abs/2505.03460v1), 2025-05-06

(21) [The Steganographic Potentials of Language Models](http://arxiv.org/abs/2505.03439v1), 2025-05-06

(22) [Procedural Memory Is Not All You Need: Bridging Cognitive Gaps in LLM-Based Agents](http://dx.doi.org/10.1145/3708319.3734172), 2025-05-06

(23) [MedArabiQ: Benchmarking Large Language Models on Arabic Medical Tasks](http://arxiv.org/abs/2505.03427v1), 2025-05-06

(24) [Directed Greybox Fuzzing via Large Language Model](http://arxiv.org/abs/2505.03425v1), 2025-05-06

(25) [Knowledge Augmented Complex Problem Solving with Large Language Models: A Survey](http://arxiv.org/abs/2505.03418v1), 2025-05-06

(26) [Lightweight Clinical Decision Support System using QLoRA-Fine-Tuned LLMs and Retrieval-Augmented Generation](http://arxiv.org/abs/2505.03406v1), 2025-05-06

(27) [Automatic Calibration for Membership Inference Attack on Large Language Models](http://arxiv.org/abs/2505.03392v1), 2025-05-06

(28) [SPAP: Structured Pruning via Alternating Optimization and Penalty Methods](http://arxiv.org/abs/2505.03373v1), 2025-05-06

(29) [Validating the Effectiveness of a Large Language Model-based Approach for Identifying Children's Development across Various Free Play Settings in Kindergarten](http://arxiv.org/abs/2505.03369v1), 2025-05-06

(30) [Geospatial Mechanistic Interpretability of Large Language Models](http://arxiv.org/abs/2505.03368v1), 2025-05-06

(31) [Interpretable Zero-shot Learning with Infinite Class Concepts](http://arxiv.org/abs/2505.03361v1), 2025-05-06

(32) [Elevating Cyber Threat Intelligence against Disinformation Campaigns with LLM-based Concept Extraction and the FakeCTI Dataset](http://arxiv.org/abs/2505.03345v1), 2025-05-06

(33) [Avoid Recommending Out-of-Domain Items: Constrained Generative Recommendation with LLMs](http://arxiv.org/abs/2505.03336v1), 2025-05-06

(34) [Absolute Zero: Reinforced Self-play Reasoning with Zero Data](http://arxiv.org/abs/2505.03335v1), 2025-05-06

(35) [AI-Driven Scholarly Peer Review via Persistent Workflow Prompting, Meta-Prompting, and Meta-Reasoning](http://arxiv.org/abs/2505.03332v1), 2025-05-06

(36) [Artificial Behavior Intelligence: Technology, Challenges, and Future Directions](http://arxiv.org/abs/2505.03315v1), 2025-05-06

(37) [Capability-Driven Skill Generation with LLMs: A RAG-Based Approach for Reusing Existing Libraries and Interfaces](http://arxiv.org/abs/2505.03295v1), 2025-05-06

(38) [Ψ-Arena: Interactive Assessment and Optimization of LLM-based Psychological Counselors with Tripartite Feedback](http://arxiv.org/abs/2505.03293v1), 2025-05-06

(39) [RAG-MCP: Mitigating Prompt Bloat in LLM Tool Selection via Retrieval-Augmented Generation](http://arxiv.org/abs/2505.03275v1), 2025-05-06

(40) [SepALM: Audio Language Models Are Error Correctors for Robust Speech Separation](http://arxiv.org/abs/2505.03273v1), 2025-05-06

(41) [Synthline: A Product Line Approach for Synthetic Requirements Engineering Data Generation using Large Language Models](http://arxiv.org/abs/2505.03265v1), 2025-05-06

(42) [SonicRAG : High Fidelity Sound Effects Synthesis Based on Retrival Augmented Generation](http://arxiv.org/abs/2505.03244v1), 2025-05-06

(43) [RobotxR1: Enabling Embodied Robotic Intelligence on Large Language Models through Closed-Loop Reinforcement Learning](http://arxiv.org/abs/2505.03238v1), 2025-05-06

(44) [DocSpiral: A Platform for Integrated Assistive Document Annotation through Human-in-the-Spiral](http://arxiv.org/abs/2505.03214v1), 2025-05-06

(45) [DYSTIL: Dynamic Strategy Induction with Large Language Models for Reinforcement Learning](http://arxiv.org/abs/2505.03209v1), 2025-05-06

(46) [A Trustworthy Multi-LLM Network: Challenges,Solutions, and A Use Case](http://arxiv.org/abs/2505.03196v1), 2025-05-06

(47) [Patterns and Mechanisms of Contrastive Activation Engineering](http://arxiv.org/abs/2505.03189v1), 2025-05-06

(48) [VLM Q-Learning: Aligning Vision-Language Models for Interactive Decision-Making](http://arxiv.org/abs/2505.03181v1), 2025-05-06

(49) [Bridging Expertise Gaps: The Role of LLMs in Human-AI Collaboration for Cybersecurity](http://arxiv.org/abs/2505.03179v1), 2025-05-06

(50) [CombiBench: Benchmarking LLM Capability for Combinatorial Mathematics](http://arxiv.org/abs/2505.03171v1), 2025-05-06

(51) [The Impact of Large Language Models on K-12 Education in Rural India: A Thematic Analysis of Student Volunteer's Perspectives](http://arxiv.org/abs/2505.03163v1), 2025-05-06

(52) [An LLM-based Self-Evolving Security Framework for 6G Space-Air-Ground Integrated Networks](http://arxiv.org/abs/2505.03161v1), 2025-05-06

(53) [Towards Effective Identification of Attack Techniques in Cyber Threat Intelligence Reports using Large Language Models](http://dx.doi.org/10.1145/3701716.3715469), 2025-05-06

(54) [Holmes: Automated Fact Check with Large Language Models](http://arxiv.org/abs/2505.03135v1), 2025-05-06

(55) [Plug-and-Play AMC: Context Is King in Training-Free, Open-Set Modulation with LLMs](http://arxiv.org/abs/2505.03112v1), 2025-05-06

(56) [Towards a standardized methodology and dataset for evaluating LLM-based digital forensic timeline analysis](http://arxiv.org/abs/2505.03100v1), 2025-05-06

(57) [Assessing and Enhancing the Robustness of LLM-based Multi-Agent Systems Through Chaos Engineering](http://arxiv.org/abs/2505.03096v1), 2025-05-06

(58) [Direct Retrieval-augmented Optimization: Synergizing Knowledge Selection and Language Models](http://arxiv.org/abs/2505.03075v1), 2025-05-05

(59) [Improving Model Alignment Through Collective Intelligence of Open-Source LLMS](http://arxiv.org/abs/2505.03059v1), 2025-05-05

(60) [Developing A Framework to Support Human Evaluation of Bias in Generated Free Response Text](http://arxiv.org/abs/2505.03053v1), 2025-05-05

(61) [34 Examples of LLM Applications in Materials Science and Chemistry: Towards Automation, Assistants, Agents, and Accelerated Scientific Discovery](http://arxiv.org/abs/2505.03049v1), 2025-05-05

(62) [Evaluating the Impact of AI-Powered Audiovisual Personalization on Learner Emotion, Focus, and Learning Outcomes](http://arxiv.org/abs/2505.03033v1), 2025-05-05

(63) [Radio: Rate-Distortion Optimization for Large Language Model Compression](http://arxiv.org/abs/2505.03031v1), 2025-05-05

(64) [UCSC at SemEval-2025 Task 3: Context, Models and Prompt Optimization for Automated Hallucination Detection in LLM Output](http://arxiv.org/abs/2505.03030v1), 2025-05-05

(65) [Memorization or Interpolation ? Detecting LLM Memorization through Input Perturbation Analysis](http://arxiv.org/abs/2505.03019v1), 2025-05-05

(66) [Modeling the Impact of Group Interactions on Climate-related Opinion Change in Reddit](http://arxiv.org/abs/2505.02989v1), 2025-05-05

(67) [Generating Narrated Lecture Videos from Slides with Synchronized Highlights](http://arxiv.org/abs/2505.02966v1), 2025-05-05

(68) [The Art of Repair: Optimizing Iterative Program Repair with Instruction-Tuned Models](http://arxiv.org/abs/2505.02931v1), 2025-05-05

(69) [RetroInfer: A Vector-Storage Approach for Scalable Long-Context LLM Inference](http://arxiv.org/abs/2505.02922v1), 2025-05-05

(70) [Scenethesis: A Language and Vision Agentic Framework for 3D Scene Generation](http://arxiv.org/abs/2505.02836v1), 2025-05-05

(71) [R1-Reward: Training Multimodal Reward Model Through Stable Reinforcement Learning](http://arxiv.org/abs/2505.02835v1), 2025-05-05

(72) [AutoLibra: Agent Metric Induction from Open-Ended Feedback](http://arxiv.org/abs/2505.02820v1), 2025-05-05

(73) [ReplaceMe: Network Simplification via Layer Pruning and Linear Transformations](http://arxiv.org/abs/2505.02819v1), 2025-05-05

(74) [Knowing You Don't Know: Learning When to Continue Search in Multi-round RAG through Self-Practicing](http://arxiv.org/abs/2505.02811v1), 2025-05-05

(75) [Towards Quantifying the Hessian Structure of Neural Networks](http://arxiv.org/abs/2505.02809v1), 2025-05-05

(76) [Generating HomeAssistant Automations Using an LLM-based Chatbot](http://arxiv.org/abs/2505.02802v1), 2025-05-05

(77) [HSplitLoRA: A Heterogeneous Split Parameter-Efficient Fine-Tuning Framework for Large Language Models](http://arxiv.org/abs/2505.02795v1), 2025-05-05

(78) [When Your Own Output Becomes Your Training Data: Noise-to-Meaning Loops and a Formal RSI Trigger](http://arxiv.org/abs/2505.02888v1), 2025-05-05

(79) [Giving Simulated Cells a Voice: Evolving Prompt-to-Intervention Models for Cellular Control](http://arxiv.org/abs/2505.02766v1), 2025-05-05

(80) [Bye-bye, Bluebook? Automating Legal Procedure with Large Language Models](http://arxiv.org/abs/2505.02763v1), 2025-05-05

(81) [Knowledge Graphs for Enhancing Large Language Models in Entity Disambiguation](http://dx.doi.org/10.1007/978-3-031-77844-5_9), 2025-05-05

(82) [FormalMATH: Benchmarking Formal Mathematical Reasoning of Large Language Models](http://arxiv.org/abs/2505.02735v1), 2025-05-05

(83) [Enhancing LLMs' Clinical Reasoning with Real-World Data from a Nationwide Sepsis Registry](http://arxiv.org/abs/2505.02722v1), 2025-05-05

(84) [Less is More: Efficient Weight Farcasting with 1-Layer Neural Network](http://arxiv.org/abs/2505.02714v1), 2025-05-05

(85) [Voila: Voice-Language Foundation Models for Real-Time Autonomous Interaction and Voice Role-Play](http://arxiv.org/abs/2505.02707v1), 2025-05-05

(86) [Structure Causal Models and LLMs Integration in Medical Visual Question Answering](http://dx.doi.org/10.1109/TMI.2025.3564320), 2025-05-05

(87) [Exploring LLM-Powered Role and Action-Switching Pedagogical Agents for History Education in Virtual Reality](http://arxiv.org/abs/2505.02699v1), 2025-05-05

(88) [AI Standardized Patient Improves Human Conversations in Advanced Cancer Care](http://arxiv.org/abs/2505.02694v1), 2025-05-05

(89) [Predicting Movie Hits Before They Happen with LLMs](http://arxiv.org/abs/2505.02693v1), 2025-05-05

(90) [Sailing AI by the Stars: A Survey of Learning from Rewards in Post-Training and Test-Time Scaling of Large Language Models](http://arxiv.org/abs/2505.02686v1), 2025-05-05

(91) [Unlearning vs. Obfuscation: Are We Truly Removing Knowledge?](http://arxiv.org/abs/2505.02884v1), 2025-05-05

(92) [A Survey on Progress in LLM Alignment from the Perspective of Reward Design](http://arxiv.org/abs/2505.02666v1), 2025-05-05

(93) [A Survey of Slow Thinking-based Reasoning LLMs using Reinforced Learning and Inference-time Scaling Law](http://arxiv.org/abs/2505.02665v1), 2025-05-05

(94) [A Note on Statistically Accurate Tabular Data Generation Using Large Language Models](http://arxiv.org/abs/2505.02659v2), 2025-05-05

(95) [Enhancing Chemical Reaction and Retrosynthesis Prediction with Large Language Model and Dual-task Learning](http://arxiv.org/abs/2505.02639v1), 2025-05-05

(96) [Parameter-Efficient Fine-Tuning with Attributed Patch Semantic Graph for Automated Patch Correctness Assessment](http://arxiv.org/abs/2505.02629v1), 2025-05-05

(97) [Detect, Classify, Act: Categorizing Industrial Anomalies with Multi-Modal Large Language Models](http://arxiv.org/abs/2505.02626v1), 2025-05-05

(98) [LLaMA-Omni2: LLM-based Real-time Spoken Chatbot with Autoregressive Streaming Speech Synthesis](http://arxiv.org/abs/2505.02625v1), 2025-05-05

(99) [Towards Cross-Modality Modeling for Time Series Analytics: A Survey in the LLM Era](http://arxiv.org/abs/2505.02583v1), 2025-05-05

(100) [EMORL: Ensemble Multi-Objective Reinforcement Learning for Efficient and Flexible LLM Fine-Tuning](http://arxiv.org/abs/2505.02579v2), 2025-05-05
